6.2 Document new Image Viewer (#480)
WoltLab/woltlab.github.io: docs/migration/wsc61/templates.md

# Migrating from WoltLab Suite 6.1 - Templates

## Image Viewer

> The previous image viewer `WCF.ImageViewer` has been open by the HTML class `.jsImageViewer`.

Reviewer: ```suggestion
The previous image viewer `WCF.ImageViewer` used the CSS class `.jsImageViewer` to open the image viewer.
```

From now on this is done via the attribute `data-fancybox`, which opens the new [Image Viewer](../../javascript/components_image_viewer.md).

> Which also now supports grouping, `data-fancybox="foo"`.

Reviewer: ```suggestion
Grouping is supported through the attribute `data-fancybox="foo"`.
```

#### Previous Code Example

```smarty
<a href="{$link}" class="jsImageViewer" title="{$title}">

>     <img src="{$link}">

Reviewer: ```suggestion
    <img src="{$thumbnailUrl}" width="…" height="…" alt="">
```

</a>
```

#### New Code Example

```smarty
<a href="{$link}" data-caption="{$title}" data-fancybox>

>     <img src="{$link}">

Reviewer: ```suggestion
    <img src="{$thumbnailUrl}" width="…" height="…" alt="">
```
